opendevin/controller/agent.py

```diff
@@ -121,3 +121,13 @@ def list_agents(cls) -> list[str]:
         if not bool(cls._registry):
             raise AgentNotRegisteredError()
         return list(cls._registry.keys())
+
+
+class AsyncAgent(Agent):
+    @abstractmethod
```

Comment:
Just a small question, this is no longer used by CodeActAgent right now, is that intended or a merge issue?

Reply:
So glad you caught this! That happened in resolving conflicts and slipped past me.
Definitely should be `AsyncAgent` here. Thanks!